crates/core/c8y_api/src/smartrest/operations.rs

```diff
@@ -77,6 +87,13 @@ impl Operations {
         let ops = ops.iter().map(|op| op.as_str()).collect::<Vec<_>>();
         declare_supported_operations(&ops)
     }
+
```

Comment:
So, as we discussed, you can add two functions here to extract the topics, one for custom JSON over MQTT handlers, and the other for SmartREST custom handlers.
This will allow you to get rid of `device_control_topics` that is added in `C8yMapperConfig`.

```rust
    pub fn get_json_custom_operation_topics(&self) -> TopicFilter {
        let mut topic_filter = TopicFilter::empty();
        for op in self.operations {
            if op.on_fragment().is_some() {
                topic_filter.add(op.topic())
            }
        }
        topic_filter
    }

    pub fn get_smartrest_custom_operation_topics(&self) -> TopicFilter {todo!()}
```